EDITORS.md

```diff
@@ -11,15 +11,11 @@ new H2 header in this file containing the instructions. -->
 
 ## Emacs Eglot
 
-[Eglot](https://github.com/joaotavora/eglot) runs solargraph server by default. To set it up with ruby-lsp you need to put that in your init file:
+[Eglot](https://github.com/joaotavora/eglot) runs solargraph server by default. To set it up with ruby-lsp you need to
+put that in your init file:
```

Comment:
```suggestion
[Eglot](https://github.com/joaotavora/eglot) runs solargraph server by default. To set it up with ruby-lsp you need to put that in your init file:
```

Reply:
(it also makes diffs much easier to read)

Reply:
I think one sentence per line might be a bit too much. As long as it's wrapping in 120 characters and it fits on the screen, it should be fine.